app/controllers/api/v1/search_controller.rb

```diff
@@ -207,6 +208,164 @@ def facet_filters
         @matching_filters = @search_facet.filters.select {|filter| filter[:name] =~ query_matcher}
       end
 
+      swagger_path '/search/auth_code' do
+        operation :post do
+          key :tags, [
+              'Search'
+          ]
+          key :summary, 'Create One-time Auth Code for downloads'
+          key :description, 'Create and return a One-Time Authorization Code to identify a user for bulk downloads'
+          key :operationId, 'search_auth_code_path'
+          response 200 do
+            key :description, 'One-time auth code and time interval, in seconds'
+            schema do
+              property :totat do
+                key :type, :integer
+                key :description, 'One-time auth code'
+              end
+              property :ti do
```

Comment:
So the reason I hadn't changed that was because the `User` model returns the fields `totat` and `ti`.  But having said that, I realize now I can manually render some JSON with the fields we agreed upon, so I'll change that.